streams.json

```diff
@@ -10,12 +10,14 @@
               "mtaylor@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/rh-messaging/artemis-wildfly-integration",
+          "repository_url": "https://github.com/rh-messaging/artemis-wildfly-integration.git",
           "codebase": "master",
           "tag": "1.0.2",
+          "group_id": "org.jboss.activemq.artemis.integration",
+          "artifact_id": "artemis-wildfly-integration",
           "version": "1.0.2",
-          "gav": "org.jboss.activemq.artemis.integration",
-          "comment": "a"
+          "gav": "artemis-wildfly-integration:org.jboss.activemq.artemis.integration:1.0.2",
+          "comment": "TAG suspicious. TAG in pom: jboss-parent-17, TAG in mapping tool: 1.0.2. a"
```

Comment:
There is no tag in the pom. Sounds like a bug in mapper.

Reply:
It looks like mapper tool is working fine here, I would say this is a bug in the pom instead. Mapper tool has inferred correctly the TAG 1.0.2, but the TAG was not in pom. Am I wrong?

Reply:
TAG in pom: `jboss-parent-17` is bogus. It depends on what are defined in POM, which are not correct in many cases.